src/chains/config/chain.config.ts

```diff
@@ -71,6 +71,9 @@ export abstract class Chain {
     if (network === Network.Polygon) {
       Chain.chains['matic'] = chain;
     }
+    if (network === Network.BinanceSmartChain) {
+      Chain.chains['binancesmartchain'] = chain;
```

Comment:
the name was updated to `binance-smart-chain` the old name has been manually added for now to support backward compatibility